engine/crates/engine-v2/src/plan/planning/collect.rs

```diff
@@ -572,7 +571,7 @@ impl<'parent, 'ctx, R: Runtime> ExecutionPlanBuilder<'parent, 'ctx, R> {
             let expected_key = if self.support_aliases {
                 self.operation_plan.response_keys.ensure_safety(field.response_key())
             } else {
-                self.operation_plan.response_keys.get_or_intern(definition.name())
+                unimplemented!("No support for resolvers that don't have alias support...");
```

Comment:
`support_aliases` thing is something I added with connectors in mind. Had to remove it here because I needed to be able to change the `ResponseKeys` (string interner) here, which is in an `Arc<Operation>`. So doesn't work anymore and left it for later 🤷